bin/README.md

```diff
@@ -47,9 +47,41 @@ $ kuzzle createFirstAdmin
 ```
 
 will guide you through the creation process of the first admin user and fix the rights to other user types if needed.
+<<<<<<< HEAD
 
 **Note:** This command is interactive and let you choose to reset the roles rights or not.
 
+# Reset Kuzzle
+
+```
+$ kuzzle likeAvirgin
+```
+
+will allow you to reset Kuzzle and restore it as if it is freshly installed.
```

Comment:
the goal to this note is to alert users that they will be asked if they want to reset the roles or not but it could be removed since the sentence above already says it